pkg/apis/kubermatic/v1/preset.go

```diff
@@ -114,6 +114,9 @@ func (s *PresetSpec) SetEnabled(enabled bool) {
 type ProviderPreset struct {
 	// Only enabled presets will be available in the KKP dashboard.
 	Enabled *bool `json:"enabled,omitempty"`
+	// IsCustomizable marks a preset as editable on the KKP UI; Customizable presets still have the credentials obscured on the UI, but other fields that are not considered private are displayed during cluster creation. Users can then update those fields, if required.
+	// NOTE: This is only supported for OpenStack Cloud Provider in KKP 2.26. Support for other providers will be added later on.
+	IsCustomizable bool `json:"isCutomizable,omitempty"`
```

Comment:
Since we have decided that this is a feature that needs to be implemented for all the cloud providers, eventually in KKP 2.27(?). It's fine to have this here as a generic option then to add it per cloud provider IMO.

Reply:
what if i want a preset to be customizable for one provider but not for another one?